src/utils.js

```diff
@@ -240,15 +241,18 @@ function intervalHandler(timerInstance) {
 	}
 
 	timerInstance.render();
+	
 	if (!timerInstance.config.duration) {
 		return;
 	}
 
 	// If the timer was called with a duration parameter,
 	// run the callback if duration is complete
 	// and remove the duration if `repeat` is not requested
+	
+	// Added new Condition of checking the timer completed or not
```

Comment:
Remove this new comment and update the comment on line 250. Change from,
`// run the callback if duration is complete`
to,
`// run the callback if duration is complete or total seconds is more than duration`